test/suites/app-service-configurable/network_test.go

```diff
@@ -0,0 +1,33 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const defaultServicePort = "59701"
+
+func TestNetworkInterface(t *testing.T) {
+	t.Cleanup(func() {
+		utils.SnapStop(t, ascService)
+	})
+
+	utils.SnapSet(t, ascSnap, "env.service.port", defaultServicePort)
```

Comment:
Why setting the port to default? Isn't the service using that port already?

Reply:
It was a mistake. Thanks! removed.